practice-app/unittests/test_books.py

```diff
@@ -0,0 +1,266 @@
+import unittest
```

Comment:
Loved the tests it is so clear that while I did the unit test I took this as a reference !